src/icrc1_auction_api.mo

```diff
@@ -315,12 +312,12 @@ actor class Icrc1AuctionAPI(quoteLedger_ : ?Principal, adminPrincipal_ : ?Princi
         r;
       },
     );
-    for ((asset, id) in Vec.items(assets)) {
-      let (symbol, decimals) = await* fetchLedgerInfo(asset.ledgerPrincipal);
-      if (asset.decimals != decimals or asset.symbol != symbol) {
-        Vec.put(assets, id, { asset with decimals; symbol });
-      };
-    };
+    // for ((asset, id) in Vec.items(assets)) {
+    //   let (symbol, decimals) = await* fetchLedgerInfo(asset.ledgerPrincipal);
+    //   if (asset.decimals != decimals or asset.symbol != symbol) {
+    //     Vec.put(assets, id, { asset with decimals; symbol });
+    //   };
+    // };
```

Comment:
Why can we comment this out?

Reply:
This code was required to the initial canister upgrade when we introduced storing decimals in the stable memory. Since we expect decimals to be never changed, it now redundant and only slows down the initialization